solidity/dashboard/src/components/TokenOverview.jsx

```diff
@@ -4,90 +4,169 @@ import * as Icons from "./Icons"
-    return percentageOf(totalOwnedStakedBalance, totalKeepTokenBalance)
-  }, [totalOwnedStakedBalance, totalKeepTokenBalance])
+  const inPercentage = useMemo(() => {
+    return percentageOf(staked, totalBalance).toString()
+  }, [staked, totalBalance])
 
-  const grantedStakedInPercentage = useMemo(() => {
-    return percentageOf(totalGrantedStakedBalance, totalGrantedTokenBalance)
-  }, [totalGrantedStakedBalance, totalGrantedTokenBalance])
+  const renderAmount = () => {
+    if (!isConnected) {
+      return "No Data"
+    } else if (isConnected && isFetching) {
+      return <Skeleton shining tag="h4" color="grey-20" width="80%" />
+    } else if (isConnected && !isFetching) {
+      return (
+        <h4 className="text-grey-40">
+          {totalBalance ? (
+            <span className="text-mint-100">
+              {displayAmountWithMetricSuffix(totalBalance)} KEEP
+            </span>
+          ) : (
+            "No data"
+          )}
+        </h4>
+      )
+    }
+  }
 
   return (
-    <section className="tile" id="token-overview-balance">
-      <div style={{ marginRight: "auto" }}>
-        <h2 className="text-grey-70">Total Balance</h2>
-        <TokenAmount amount={totalKeep} withMetricSuffix />
-        <h3 className="text-grey-30">{`${totalStakedInPercentage}% Staked`}</h3>
-      </div>
-      <div
-        style={boxWrapperStyle}
-        className="flex column center space-between mt-1"
-      >
-        <Icons.GrantContextIcon />
-        <h4 className="text-grey-70">Granted Tokens</h4>
-        <TokenAmount
-          currencyIconProps={{ width: 18, heigh: 18 }}
-          amountClassName="h4 text-primary"
-          suffixClassName="text-small text-primary"
-          amount={totalGrantedTokenBalance}
-          withMetricSuffix
+    <section className={`balance__${type}`}>
+      <header className="flex row center mb-2">
+        <Chip
+          icon={<IconComponent />}
+          color="disabled"
+          className={`balance__${type}__chip`}
         />
-        <p className="text-small">{`${grantedStakedInPercentage}% Staked`}</p>
-        <Link
-          to={{ pathname: "/tokens/delegate", hash: "#granted" }}
-          className="btn btn-primary mt-2"
-        >
-          manage
-        </Link>
-      </div>
-      <div
-        style={boxWrapperStyle}
-        className="flex column center space-between mt-1"
+        <div className="flex-1">
+          <h3 className="text-grey-70 flex row center">
+            {type}&nbsp;
+            {tooltipProps && <SpeechBubbleTooltip {...tooltipProps} />}
+          </h3>
+          {renderAmount()}
+        </div>
+      </header>
+      <ProgressBar
+        items={[{ value: staked, color: colors.primary }]}
+        total={totalBalance}
+        styles={{ margin: 0 }}
+      />
+      <span className="text-small text-grey-40">
+        {isConnected ? inPercentage : "-"}% Staked
+      </span>
+      <Link
+        to={{ pathname: "/delegate", hash: type }}
```

Comment:
This link, and the other ones to staking from the overview, seems to be broken. Let's fix it before merging things to master.

Reply:
fixed, pls see https://github.com/keep-network/keep-core/pull/2113/commits/0c060205b502b06f6c10551fcd6873cc55be082f